src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -22,15 +23,39 @@ function getActionText(reportAction) {
     return lodashGet(message, 'html', '');
 }
 
+export const CONTEXT_MENU_TYPES = {
+    LINK: 'LINK',
+    REPORT_ACTION: 'REPORT_ACTION',
+};
+
 // A list of all the context actions in this menu.
 export default [
-    // Copy to clipboard
     {
-        textTranslateKey: 'contextMenuItem.copyToClipboard',
+        textTranslateKey: 'reportActionContextMenu.openLink',
```

Comment:
Let's actually get rid of this context action. It's just not useful ... if you want to go to the link you can just click it, or CMD+click to open in a new tab. That means we can get rid of the `reportActionContextMenu.openLink` translation too.